ethexe/runtime/src/wasm/api/mod.rs

```diff
@@ -46,13 +46,18 @@ fn _run(arg_ptr: i32, arg_len: i32) -> i64 {
     let (program_id, original_code_id, state_root, maybe_instrumented_code) =
         Decode::decode(&mut get_slice(arg_ptr, arg_len)).unwrap();
 
-    let res = run::run(
+    let journal = run::run(
         program_id,
         original_code_id,
         state_root,
         maybe_instrumented_code,
     );
 
+    let chunks = journal.encoded_size() / 32 * 1024 * 1024 + 1; // never zero
+    let chunk_size = (journal.len() / chunks).max(1); // never zero
```

Comment:
Better to panic in case it is zero because in other case we would panic in allocator